tools/depends/target/ffmpeg/CMakeLists.txt

```diff
@@ -37,6 +56,13 @@ elseif(CORE_SYSTEM_NAME STREQUAL osx)
                           --disable-decoder=mpeg_xvmc --disable-vda --disable-crystalhd --enable-videotoolbox
                           --target-os=darwin
                           --disable-securetransport)
+elseif(CORE_SYSTEM_NAME STREQUAL rbpi)
+  if(CPU STREQUAL "cortex-a7" OR CPU STREQUAL "cortex-a53")
+    list(APPEND ffmpeg_conf --cpu=${CPU} --enable-pic --disable-armv5te --disable-armv6t2)
+  endif()
+  list(APPEND ffmpeg_conf --enable-hardcoded-tables --disable-vaapi --disable-vdpau --enable-mmal
+                          --enable-omx-rpi)
+
```

Comment:
FFT is widely used on decompressing, so there might be a benefit there. If it is noticeable, I also have some doubts. 
I'll do the following, I'll edit this PR and remove the hardcoded-tables (along with the others you mentioned previously), and I'll do some testing. If it proves to be noticeable I'll open a new PR to add that option.

Thanks for taking the time to review this PR.